.script/compileSC.sh

```diff
@@ -1,7 +1,61 @@
 #!/bin/bash
 
-solc --abi --bin contracts/LiquidStakingV2.sol -o build
-abigen --bin ./build/LiquidStakingV2.bin --abi ./build/LiquidStakingV2.abi --pkg=liquidStaking --out=liquidStaking.go
+OPENZEPPELIN_DIR_NAME="@openzeppelin"
 
-solc --abi --bin contracts/TokenWrapperV2.sol -o build --overwrite
-abigen --bin ./build/TokenWrapperV2.bin --abi ./build/TokenWrapperV2.abi --pkg=tokenWrapper --out=TokenWrapper.go
+rm -rf pStake-smartContracts
+rm -rf "$OPENZEPPELIN_DIR_NAME"
+rm -rf ./ethereum/abi/liquidStaking/liquidStaking.go
+rm -rf ./ethereum/abi/tokenWrapper/tokenWrapper.go
+
+echo "Downloading pStake-smartContracts..."
+git clone git@github.com:persistenceOne/pStake-smartContracts.git
```

Comment:
Could we clone only the master branch? (here and below in the script)

Reply:
Different version of SCs are used for different chains, xprt and atom have little different released versionss

Reply:
I was more about cloning only master/main branch without unnecessary remote branches.